docs/pages/model_hub/asr.md

```diff
@@ -49,4 +49,68 @@ Here are some other possible configurations for the Whisper deployment:
             compute_type=WhisperComputeType.FLOAT32,
         ).model_dump(mode="json"),
     )
-    ```
\ No newline at end of file
+    ```
+
+### Diarized ASR
+
+Diarized transcription can be generated by using [WhisperDeployment](./../../reference/deployments.md#aana.deployments.WhisperDeployment) and [PyannoteSpeakerDiarizationDeployment](./../../reference/deployments.md#aana.deployments.PyannoteSpeakerDiarizationDeployment) and combining the timelines using post processing with [ASRPostProcessingForDiarization](./../../reference/processors.md).
+
+Example configuration for the PyannoteSpeakerDiarization model is available at [Speaker Diarization](./speaker_recognition.md) model hub.
+
+You can simply define the model deployments and the endpoint to transcribe the video with diarization. Below code snippet shows the custom endpoint class `TranscribeVideoWithDiarEndpoint` to combine the outputs from ASR and diarization deployments:
+
+    ```python
```

Comment:
The code is not displayed properly in the docs, probably because of the extra indentation. Please start docs with `poetry run mkdocs serve` to see if it displays properly.

Reply:
Also can we boil it down to the basics, just to show how to produce and combine ASR and SD segments, and refer to notebook for detailed example?

Reply:
Removed the class and just retained the combination logic